masterlist.yaml

```
@@ -20511,6 +20524,9 @@ plugins:
         type: warn
         subs: [ '[Frostcrag Alchemy Sorter Updated](https://www.nexusmods.com/oblivion/mods/49969/)' ]
         condition: 'version("FrostcragAlchemySorter.esp", "1.0", <)'
+      - <<: *alreadyInX
+        subs: [ '[Frostcrag Spire Coblized](https://www.nexusmods.com/oblivion/mods/52608/)' ]
+        condition: 'active("DLCFrostcragCoblAlchemyTable.esp")'
```

```suggestion
        subs: [ '**DLCFrostcragCoblAlchemyTable.esp**' ]
        condition: 'active("DLCFrostcragCoblAlchemyTable.esp") and version("DLCFrostcragCoblAlchemyTable.esp", "2.0", >=)'
```

Same as above. This message should also be sorted above the `patchUpdateAvailable` message (I can't do it in a suggestion).